### ISSUE TITLE
DNN Unknown layer type Mean

### ISSUE THREAD

**Issue by Jaykob** [opencv/opencv_contrib#1350]
**Accidentally posted in contrib, sorry! Can be deleted here!**
  
  
  

- OpenCV => 3.3.0 (current master)
- Operating System / Platform => macOS 10.12.6
- Compiler => clang

##### Detailed description

I'm trying to load a relatively simple tensorflow model which uses batch norm with the dnn module. and I get the following error.
I also applied PR #9517 but that didn't change anything.
Is there something I'm missing or any hint what I could do to successfully import that model?

```
OpenCV Error: Unspecified error (Unknown layer type Mean in op conv1/conv1/moments/Mean) in populateNet, file /Users/jaykob/opencv/modules/dnn/src/tensorflow/tf_importer.cpp, line 960
/Users/jaykob/opencv/modules/dnn/src/tensorflow/tf_importer.cpp:960: error: (-2) Unknown layer type Mean in op conv1/conv1/moments/Mean in function populateNet
```
